sample_flask_code.py

```diff
@@ -0,0 +1,63 @@
+
+from flask import Flask,request,jsonify
+import json 
+import os
+
+app=Flask(__name__)
+
+data_file='data.json'
+
+try:
+     with open(data_file, 'r') as file:
+        tasks = json.load(file)
+            
+except FileNotFoundError:
+    with open(data_file, 'w') as file:
+     tasks = []
+     json.dump(tasks, file)
+
+@app.route('/www.files.com/', methods=['GET'])
+def get_tasks():
+    try:
+        with open(data_file, 'r') as file:
+            tasks = json.load(file)
+        return jsonify(tasks)
+    except Exception as e:
+        return jsonify({"error": str(e)}), 500
+
+
+@app.route('/www.files.com/', methods=['POST'])
+def create_task():
+    try:
+        new_task = request.get_json()
+
+        with open(data_file, 'r') as file:
+            tasks = json.load(file)
+
+        new_task['id'] = len(tasks) + 1
```

Comment:
This way of generating id is wrong.
Consider a case, When you have items having id 7 and 8.
And I delete the item with id 7
And when you create new task, Then it will also have id 8, which already exists